src/icons/Bell Icon.svg

```diff
@@ -1,28 +1,28 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
-	 x="0px" y="0px" width="24px" height="24px" viewBox="0 0 24 24"
-	 style="enable-background:new 0 0 24 24;" xml:space="preserve">
-<style type="text/css">
-	.st0{fill:#595959;}
-</style>
-<g id="XMLID_51_">
-	<g id="XMLID_59_">
-		<path id="XMLID_61_" class="st0" d="M2.4,11.4C2.4,11.4,2.4,11.4,2.4,11.4c-0.5,0-0.9-0.4-0.9-0.9c0-0.2,0.1-4.1,3.8-6.3
-			c0.4-0.3,1-0.1,1.2,0.3c0.3,0.4,0.1,1-0.3,1.2c-2.8,1.7-2.9,4.6-2.9,4.8C3.3,11,2.9,11.4,2.4,11.4z"/>
-	</g>
-	<g id="XMLID_57_">
-		<path id="XMLID_58_" class="st0" d="M21.6,11.4c-0.5,0-0.9-0.4-0.9-0.9c0-0.1-0.1-3.1-2.9-4.8c-0.4-0.3-0.6-0.8-0.3-1.2
-			c0.3-0.4,0.8-0.6,1.2-0.3c3.6,2.2,3.8,6.1,3.8,6.3C22.5,10.9,22.1,11.4,21.6,11.4C21.6,11.4,21.6,11.4,21.6,11.4z"/>
-	</g>
-	<path id="XMLID_52_" class="st0" d="M21.3,18.7c-1.4-1.4-2.2-3.3-2.2-5.3v-1.2c0-0.3,0-0.6-0.1-0.9c-0.2-2.1-1.4-4.1-3.1-5.3
-		C15.6,6,15.3,5.8,15,5.6l-0.3-0.2l0.2-0.3c0.3-0.5,0.5-1.2,0.5-1.8C15.4,1.5,13.9,0,12,0S8.6,1.5,8.6,3.4c0,0.6,0.2,1.2,0.5,1.8
-		l0.2,0.3L9,5.6c-0.3,0.2-0.7,0.3-1,0.5C6.7,7,5.8,8.3,5.3,9.8c-0.2,0.7-0.4,1.4-0.4,2.2v1.4c0,2-0.8,3.9-2.2,5.3
-		c-0.3,0.3-0.3,0.7-0.2,1c0.1,0.3,0.4,0.6,0.8,0.6h4.9h0.3v0.3c0,1.9,1.5,3.4,3.4,3.4s3.4-1.5,3.4-3.4v-0.3h0.3h4.9
-		c0.4,0,0.7-0.3,0.8-0.6C21.6,19.5,21.6,19.1,21.3,18.7z M12,1.8c0.9,0,1.6,0.7,1.6,1.6S12.9,4.9,12,4.9s-1.6-0.7-1.6-1.6
-		C10.5,2.5,11.1,1.8,12,1.8z M13.1,21.7c-0.3,0.3-0.7,0.5-1.1,0.5c-0.9,0-1.6-0.7-1.6-1.6v-0.3h0.3h2.5h0.3l0,0.3
-		C13.6,21,13.4,21.4,13.1,21.7z M14.8,11c0,0.3-0.1,0.6-0.4,0.9l-2.9,3.2h2.6c0.4,0,0.7,0.3,0.7,0.7s-0.3,0.7-0.7,0.7h-4
-		c-0.5,0-0.9-0.3-0.9-0.7v0c0-0.3,0.1-0.6,0.4-0.9l2.9-3.2H10c-0.4,0-0.7-0.3-0.7-0.7c0-0.4,0.3-0.7,0.7-0.7h3.9
-		C14.4,10.3,14.8,10.6,14.8,11L14.8,11z"/>
-</g>
-</svg>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Hmm, so in Dev Edition, the disabled icon is lighter and the enabled icon is darker. That made me think the states were switched. But, this sounds like a variation on Issue #67. 

Maybe we want to think about changing the icon shape itself rather than the color? (i.e. circle-with-crossbar over the bell? just a random thought)